dataeng/resources/tableau-backup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Would you mind adding a comment into the top of this script describing what it does?

Reply:
Sure, absolutely